target/linux/mvebu/image/cortexa9.mk

```diff
@@ -448,3 +448,20 @@ define Device/synology_ds213j
     -ppp -kmod-nft-offload -dnsmasq -odhcpd-ipv6only
 endef
 TARGET_DEVICES += synology_ds213j
+
+define Device/wd_cloud-gen2
+  DEVICE_VENDOR := Western Digital
+  DEVICE_MODEL := MyCloud Gen 2 (WD_Glacier)
+  DEVICE_PACKAGES += -uboot-envtools mkf2fs e2fsprogs badblocks \
+	partx-utils kmod-hwmon-drivetemp kmod-usb-storage \
```

Comment:
`CONFIG_USB_STORAGE` is enabled by default and `kmod-usb-storage` can be removed.

https://github.com/openwrt/openwrt/blob/207bfee855320dc938f39e179b1d2e3b008140cb/target/linux/mvebu/config-6.6#L426